static/About.html

```diff
@@ -0,0 +1,39 @@
+<html>
+<head>
+<title>
+  Just a Ruby Girl, In a Ruby World...</title>
+</head>
+<header>
+  <a href="/home">Home</a>
+  <a href="/about">About</a>
+  <a href="/blog">Blog</a>
+  <a href="/projects">Projects</a>
+</header>
+<body>
+<p>
```

Comment:
This looks pretty good. :) Only thing missing is the closing tag for your `<p>`s. The majority of html tags come in pairs. For every opening tag, there's a closing tag that defines where it ends. Your header, for example, is perfectly structured; each link (`<a>` tag) wraps some text, and they are all wrapped by the `<header>` tag.

Your `<p>`s work the same way. Use them to group text (and sometimes images) together into paragraphs, both visually and in terms of the document structure.